packages/tokens/src/tokens.scss

```diff
@@ -243,11 +243,11 @@
     --text-caption-letter: 0.004em;
     --text-caption-focused: var(--font-weight-regular) 12px/20px
       var(--font-family);
-    --text-caption-focused-letter: 0.0004em;
+    --text-caption-focused-letter: 0.004em;
     --text-overline: var(--font-weight-regular) 11px/20px var(--font-family);
-    --text-overline-letter: 0.014em;
+    --text-overline-letter: 0.16em;
     --text-badge: var(--font-weight-bold) 10px/16px var(--font-family);
-    --text-badge-letter: 0.0004em;
```

Comment:
is this --text-badge-letter right too? The diff is bigger than others

Reply:
Yes, it is the example in the evidence. According to the [Styleguide](https://www.figma.com/design/rC8E8nEOrESRPoQQGt4kSx/Styleguide-2.0?node-id=410-0&t=ldVZqQZhcAegWtf6-0), the value is 4%, which is equal to `0.04em`<img width="641" alt="image" src="https://github.com/user-attachments/assets/4c08c490-323f-4a2e-bfe9-987f8548ea98">